application/org.openjdk.jmc.flightrecorder.flameview/src/main/resources/page.template

```diff
@@ -29,7 +29,7 @@
 			flameGraph = d3.flamegraph()
 				.width(windowSize() * 0.9)
 				.cellHeight(18)
-				.transitionDuration(500)
+				.transitionDuration(320)
```

Comment:
Was this change intended to be included in this formatting PR?

Reply:
"Also lowering the animation time for animations, since they are annoyingly slow now." 
Yes, thought I might as well, since it's such a small change.

Reply:
it makes sense

Reply:
it makes sense